doc/examples/imapd_conf/.gitignore

```diff
@@ -0,0 +1 @@
+imapd.conf.sample
```

Comment:
I'd prefer if this was in the top-level .gitignore rather than buried down in a subdirectory -- makes it easier to see all the ignored files in one place -- but it's not a big deal.  Looking around, I see there's a few other directory-specific .gitignore files lying around, so I might just do a big cleanup of all of them sometime.